src/ui/public/chrome/index.d.ts

```diff
@@ -17,8 +17,13 @@
  * under the License.
  */
 
+interface IInjector {
```

Comment:
The `auto` namespace is not exported, so unfortunately we can't use that typing (I have no idea why the typing doesn't export that type, since it's a legit service to inject).

Reply:
good point